packages/traceability/issuer-api/src/pods/certificate/commands/batch-transfer-certificates.command.ts

```diff
@@ -0,0 +1,10 @@
+import { BigNumber } from 'ethers';
+
+export class BatchTransferCertificatesCommand {
+    constructor(
+        public readonly certificateIds: number[],
+        public readonly to: string,
```

Comment:
So I have to transfer all certificates from one address to another? Isn't it possible to transfer different certificates to different addresses in one batch?

It will make fix complicated on implementation side, because certificates would have to additionally grouped into batches by this address.